dashbuilder/dashbuilder-backend/dashbuilder-services/src/main/java/org/dashbuilder/transfer/DataTransferServicesImpl.java

```diff
@@ -181,19 +198,39 @@ public FileVisitResult visitFile(Path path, BasicFileAttributes attrs) {
             }
         });
 
-        ioService.deleteIfExists(
-                root.resolve(expectedPath));
+        ioService.deleteIfExists(root.resolve(expectedPath));
 
         return imported;
     }
 
-    private List<String> importFiles(Path path) throws Exception  {
-        String tempPath = new StringBuilder()
-            .append(System.getProperty("java.io.tmpdir"))
-            .append(File.separator)
-            .append(FILE_PATH)
-            .append(File.separator)
-            .toString();
+    @Override
+    public DataTransferAssets assetsToExport() {
+        List<String> pages = listPaths(perspectivesFS,
+                                       p -> p.endsWith(FILE_EXT)).stream()
+                                                                 .map(p -> p.getName(p.getNameCount() - 2))
+                                                                 .map(Object::toString)
+                                                                 .collect(Collectors.toList());
+
+        List<DataSetDef> datasetsDefs = listPaths(datasetsFS,
+                                                  p -> p.endsWith(DATASET_EXT)).stream()
+                                                                               .map(this::getFileContent)
+                                                                               .map(this::parseDataSetDefinition)
+                                                                               .collect(Collectors.toList());
+        return new DataTransferAssets(datasetsDefs, pages);
+    }
+
+    @Override
+    public Map<String, String> validateExport(DataTransferExportModel exportsModel) {
+        // TODO Auto-generated method stub
+        return null;
```

Comment:
This method was removed, I would try to send this PR implementing two JIRAs, but it would be too much work for a PR review, but I forgot to remove this method.